src/KubernetesClient/Informers/Controller.cs

```diff
@@ -0,0 +1,122 @@
+using k8s.cache;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace k8s.informers
+{
+    public class Controller<T, L> : IController<T, L>
+    {
+        public IReflector<T,L> _reflector;       
+        public IStore _store;
+        private Queue<Tuple<WatchEventType, IKubernetesObject>> _fifo;
+        private AutoResetEvent _fifoEvent;
+        private Action<WatchEventType, IKubernetesObject> _onAdd;
```

Comment:
Shouldn't these be `List<Action<...>>` ?

Reply:
As in multiple Actions in case of add, delete and update ? Sure, can accommodate that requirement.